testing/ef_tests/src/cases/ssz_generic.rs

```diff
@@ -326,6 +326,6 @@ where
             N::to_usize()
         )))
     } else {
-        Ok(decoded.into())
+        Ok(decoded.try_into().unwrap())
```

Comment:
Could probably remove the `unwrap` here and simplify, by combining the error case (`Too many values`) with the `try_into`. Something like `try_into().map_err(...)` and get rid of the manual check